cloverage/test/cloverage/instrument_test.clj

```diff
@@ -267,46 +267,10 @@
                        getName))
                (rw/macroexpand-all (inst/instrument-form #'inst/nop nil '(.getName (let [x 1]
                                                                                      (str "X is" x)
-                                                                                     (Thread/currentThread))))))))))
-
-(t/deftest instrument-inlined-primitives-test
-  (t/testing "Inline primitive cast functions like int() should be instrumented correctly (#277)"
-    (t/is (= '(. clojure.lang.RT (intCast 2))
-             (rw/macroexpand-all (inst/instrument-form #'inst/no-instr nil '(int 2)))
-             (rw/macroexpand-all (inst/instrument-form #'inst/no-instr nil `(int 2)))))
-    (t/testing "make sure example in #277 actually gets instrumented and doesn't fall back to returning the original form"
-      (let [form '(deftype MyType [^:unsynchronized-mutable ^int i]
```

Comment:
Wait, so, this test was.. wrong? I feel like there's a regression test in here we're losing, can you (briefly) convince me otherwise?

Reply:
Sorry @lvh, I did not mean to delete that test at all. I'm not sure how I ended up doing that. Let me fix the PR

Reply:
I think I must have done branched off an old version of master that didn't have those tests in there yet. Fixed